modules/RTC/ScreenObtainer.js

```diff
@@ -163,9 +163,18 @@ const ScreenObtainer = {
             getDisplayMedia = navigator.mediaDevices.getDisplayMedia.bind(navigator.mediaDevices);
         }
 
+        const { disableAGC, disableAP, enableHdAudio } = this.options;
```

Comment:
`enableHdAudio` is not something LJM should be concerned with, we already pass the individual options from the app.

Reply:
Yeah, was being lazy :P Will base it on `stereo`